website/project/model.py

```diff
@@ -2359,15 +2371,19 @@ def add_log(self, action, params, auth, foreign_user=None, log_date=None, save=T
             user=user,
             foreign_user=foreign_user,
             params=params,
+            node=self,
+            original_node=params['node']
         )
 
         if log_date:
             log.date = log_date
+        log.save()
 
-        self.date_modified = log.date.replace(tzinfo=None)
+        if len(self.logs) == 1:
```

Comment:
I hate to keep tagging on this line but I'm really not understanding why the date would originally be incorrect thus the `if/else` being necessary...? Also if this is needed please add a comment as this seems like its saying, if we have `1` log, grab it and get the date, if we have more than one log just grab the last one and use its date, which is the same as if we have one log right... :confused: